setup.php

```diff
@@ -82,7 +82,7 @@ public function enqueue_scripts($page_hook) {
     $current_screen = get_current_screen();
     $current_post_type = get_post_type_object($current_screen->post_type);
 
-    if($current_post_type->public === false) return;
+    if( $this->check_if_yoast_seo_is_hidden($current_post_type->name) === true ) return;
```

Comment:
I need to put together some code style preference for this seeing as it's now getting some external PR's 😉 

Could we format this line as (stripping unnecessary whitespace effectively):

```
 if($this->check_if_yoast_seo_is_hidden($current_post_type->name) === true) return;
```